ts/features/walletV3/onboarding/utils/index.ts

```diff
@@ -13,8 +16,21 @@ import { isStringNullyOrEmpty } from "../../../../utils/strings";
 // List of outcomes that are considered successful
 export const successOutcomes: ReadonlyArray<OnboardingOutcome> = ["0"];
 
+export const ONBOARDING_FAR_ENABLE_3DS = "https://io.italia.it/faq/#n3_3";
```

Comment:
```suggestion
export const ONBOARDING_FAQ_ENABLE_3DS = "https://io.italia.it/faq/#n3_3";
```
I'm guessing its a typo, correct?

Reply:
Yes, it's a typo, thanks!